prow/plugins.yaml

```diff
@@ -222,6 +222,29 @@ repo_milestone:
     maintainers_id: 2460384
     maintainers_team: kubernetes-milestone-maintainers
 
+project_config:
+  project_org_configs:
+    kubernetes:
+      org_maintainers_team_id: 2460384
+      org_maintainers_team_name: kubernetes-milestone-maintainers
+      org_default_column_map:
+        1.14 CI Signal:
+          Start here
+        KEP Implementation Tracking:
+          To do
+      project_repo_configs:
+        kubernetes:
+          repo_maintainers_team_id: 2460384
```

Comment:
Do we need to specify these? They are the same as in the owning org

Reply:
It's probably a bad example for me to use the same team here. They could be different and the repo setting will override the org setting